starknet_py/net/full_node_client.py

```diff
@@ -93,6 +95,47 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
+        address: Optional[Hash] = None,
+        keys: Optional[List[Hash]] = None,
+        continuation: bool = False,
+        chunk_size: int = 1,
+    ) -> Tuple[Events, Optional[str]]:
+        # pylint: disable=too-many-arguments
+        params = {
+            "chunk_size": chunk_size if chunk_size >= 1 else 1,
+            "from_block": get_block_identifier(from_block_hash, from_block_number),
+            "to_block": get_block_identifier(to_block_hash, to_block_number),
+        }
+
+        if address is not None:
+            params["address"] = _to_rpc_felt(address)
+        if keys is not None:
+            params["keys"] = list(map(_to_rpc_felt, keys))
+
+        res = await self._client.call(
+            method_name="getEvents",
+            params={"filter": params},
+        )
+        events = cast(Events, EventsSchema().load(res, unknown=EXCLUDE))
+        continuation_token = res.get("continuation_token")
```

Comment:
I think it should be done at schema level as well. Let's add continuation token to `EventsSchema` as non required field with default of None.